stack.go

```diff
@@ -131,6 +131,8 @@ type StackConfiguration struct {
 	ErrorMessage         *string                             `jsonapi:"attr,error-message"`
 	EventStreamURL       string                              `jsonapi:"attr,event-stream-url"`
 	Diagnostics          []*StackDiagnostic                  `jsonapi:"attr,diags"`
+
+	Relationships map[string]interface{} `json:"relationships"`
```

Comment:
Rather using a `map[string]interface{}` for the relationships, we typically denote the specific item in the relationship. Then in the `jsonapi` tags we denote it as coming from the relationship. You can look at the `StackDeployment` struct below to get an idea on how that looks. The only present relationship here will be the configuration's Stack.